.circleci/config.yml

```diff
@@ -446,6 +446,9 @@ workflows:
             - docs-verdaccio
       - comment:
           name: comment-pr
+          filters:
+            branches:
+              ignore: main
```

Comment:
derp, for some reason I thought this flow was only happening on PRs, fixed so it now only runs when not on main

Reply:
and the double comment key?

Reply:
https://github.com/adobe/react-spectrum/pull/2432#discussion_r947042637